### PR TITLE
Add coded to asmt table.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: reporting_olap/sql/V1_0_0_0__ddl.sql [hunk 1]
```diff
@@ -245,6 +245,7 @@ CREATE TABLE state_embargo (
 CREATE TABLE asmt (
   id int encode raw NOT NULL PRIMARY KEY,
   grade_id smallint NOT NULL,
+  grade_code character varying(2) NOT NULL,
```
Comment (agorina) [inline reporting_olap/sql/V1_0_0_0__ddl.sql]: People really like codes...

Comment (malaffoon) [reply reporting_olap/sql/V1_0_0_0__ddl.sql]: but only for grade, not subject, type, etc?

Comment (agorina) [reply reporting_olap/sql/V1_0_0_0__ddl.sql]: Good point. If I add `subject_code`, I can remove one more JOIN. Will do.

**Comment by agorina** [comment]
This was not such a good idea.

Just to explain why: the report SQL uses ids in WHERE and GROUP BY. By adding codes, we now have to add them to the GROUP BY, that leads to changes to the SORT KEY.

After talking with @wtritch , I am going to abandon this change and will do translation in Java.

**Pull request closed without merging**